tests/console/yast2_apparmor.pm

```diff
@@ -123,17 +136,22 @@ sub run {
     type_string '/new_profile';
     wait_screen_change { send_key 'alt-o' };
 
-    # check profile dialog after new profile created and add a new rule to it
-    assert_screen 'yast2_apparmor_configuration_profile_dialog';
-    send_key 'alt-a';
-    wait_screen_change { send_key 'alt-r' };
-    wait_screen_change { send_key 'alt-s' };
-    send_key 'alt-d';
-
-    # confirm to save changes to profile and finish test
-    assert_screen 'yast2_apparmor_configuration_manage_existing_profiles_edit_file_changed_again';
-    send_key 'alt-y';
-    wait_serial("yast2-apparmor-status-0", 200) || die "'yast2 apparmor' didn't finish";
+    if (!(sle_version_at_least('15'))) {
+        # check profile dialog after new profile created and add a new rule to it
+        assert_screen 'yast2_apparmor_configuration_profile_dialog';
+        send_key 'alt-a';
+        wait_screen_change { send_key 'alt-r' };
+        wait_screen_change { send_key 'alt-s' };
+        send_key 'alt-d';
+  
+        # confirm to save changes to profile and finish test
+        assert_screen 'yast2_apparmor_configuration_manage_existing_profiles_edit_file_changed_again';
+        send_key 'alt-y';
+        wait_serial("yast2-apparmor-status-0", 200) || die "'yast2 apparmor' didn't finish";
+    }
+    else {
+        send_key 'alt-o';
```

Comment:
I suggest to make that behaviour an early return, e.g.

```perl
if (…) {
    send_key 'alt-o';
    return;
}
…

Reply:
There is literally nothing afterwards, in the best case this would just improve the readability and then I would also put it in the if branch to keep it unified

Reply:
uhm, don't get it. Whatever :)